persistence/pre_persistence/proto/account.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package utility;
+option go_package = "/utility/test/";
+
+message Account {
+  bytes Address = 1;
```

Comment:
I realize that in golang, we use CamelCase and capitalize exported fields, but protobuf style guide prefers snake case: https://developers.google.com/protocol-buffers/docs/style

In addition, I confirmed locally that if you name an attribute `address_foo`, it translates to `AddressFoo` when `protoc` generates the Go structs.

Reply:
I assume there's no actionable here in this PR. Something to investigate

Reply:
The actionable would be to change `bytes Address = 1;` to `bytes address = 1;` in the *.proto files and the *.go files would not need to be changed.

Reply:
Okay makes sense

Reply:
Going to use my script to update these myself locally.